### DISCUSSION TITLE
Welcome to dreambooth-gui Discussions!

### DISCUSSION THREAD

**Discussion by smy20011** [smy20011/dreambooth-gui#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZeroCool22** [comment]
![giphy](https://user-images.githubusercontent.com/13344308/200003492-411f8e72-938e-45ad-9b6c-74848a5d2391.gif)

**Comment by bmoggedhwm** [comment]
got to say thank you for your good work I can't even get the damn thing to work under windows. Making a gui for it is the next level.

**Comment by montcc** [comment]
Thank you for your work, but I ran into an error. I don't know why------------------------------------docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: exit status 1, stdout: , stderr: Auto-detected mode as 'legacy'
nvidia-container-cli: initialization error: WSL environment detected but no adapters were found: unknown.

**Comment by QiNuoTu** [comment]
I successfully deployed it according to the instructions in the document, but the following prompt appears when training according to the methods in the document

Unable to find image 'smy20011/dreambooth:latest' locally

latest: Pulling from smy20011/dreambooth

I have specified the path of the image file, and the image set is correctly displayed in the UI. I don't understand why the prompt that I can't find an image appears, and it is not any folder I have specified. Many of my friends have encountered the same problem

**Comment by fumitakayano** [comment]
Thank you for your great work.
A 10GB+ GPU runs dreambooth and comes with a gui.
I am thrilled!

If there's anything you've planned on your roadmap going forward, or if there's anything you think is feasible, I'd love to hear from you.

I have a Quadro RTX 4000 on my laptop, is it possible to run dreambooth-gui on a 8GB+ GPU and not get the error?
If there is a possibility, please try it!
